js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -200,32 +249,32 @@
 			});
 		};
 
+		// GOOD
 		const processAndCombineEvents = (eventResults) => {
 			vm.isLastPage = isLastPage(eventResults.totalResults);
 			vm.eventGroups = combineEventGroups(vm.eventGroups, eventResults.eventGroups);
 		};
 
+		// GOOD
 		const isLastPage = (totalResults) => {
-			const totalResultsSoFar = requestModel.Page * vm.chunkSize;
+			const totalResultsSoFar = vm.requestModel.Page * vm.chunkSize;
 			return totalResultsSoFar >= totalResults;
 		};
 
+		// GOOD
 		const isDateRangeValid = (firstDate, secondDate) => {
 			if (firstDate && secondDate) {
 				return $window.moment(firstDate).isSameOrBefore(secondDate);
 			}
-
 			return false;
 		};
 
-		const isSameDay = (day1Date, day2Date) => {
-			if (day1Date && day2Date) {
-				return $window.moment(day1Date).isSame(day2Date, 'day');
-			}
-
-			return false;
-		};
+		// GOOD
+		const isSameDay = (day1Date, day2Date) => day1Date && day2Date ?
+			$window.moment(day1Date).isSame(day2Date, 'day') :
+			false;
```

Comment:
Beautiful refactor.  Because this already spans multiple lines, I might help clarify that this is a function by separating the inputs from the expression.

```
const isSameDay = (day1Date, day2Date) =>
  day1Date && day2Date ?
    $window.moment(day1Date).isSame(day2Date, 'day') :
    false;
```

Reply:
@ascire i don't think this code was actually modified, just moved, wasn't a case for a refactor.

Reply:
It's the exact same function as on the left, just converted to a lambda expression, which is good since it has no side effects in the first place.

Reply:
@ascire I get the update, I'm just saying that function wasn't touched as part of "Events Fixes" as described above, I think it's showing in green because it was moved or copied in.